posthog/api/test/test_query_ws.py

```diff
@@ -0,0 +1,98 @@
+from unittest.mock import patch
+from freezegun import freeze_time
+from channels.testing import WebsocketCommunicator
+from django.test import TransactionTestCase
+from posthog.rate_limit import ClickHouseSustainedRateThrottle
+from asgiref.sync import sync_to_async
+from posthog.models.organization import Organization
+from posthog.models.team import Team
+from posthog.models.user import User
+from posthog.models.project import Project
+import random
+
+
+from posthog.asgi import application
+
+from typing import Optional
+
+import functools
```

Comment:
nit: These should be sorted. Maybe you are missing an IDE config to do it for you on save and/or commit?